proposals/052-cryptography-leg-1.md

```diff
@@ -0,0 +1,393 @@
+# Botan Cryptography Community Project Proposal
+
+# Abstract
+
+This community project proposal is for full-time funding for the development of a suite of Haskell cryptography libraries and tooling suitable a wide range of uses including data integrity, privacy, security, and networking. This is necessary to relieve the burden [of correctly implementing cryptography] from Haskell developers seeking to provide privacy and security to their users. We propose the development of a community-owned set of cryptography libraries of increasing capability, starting with bindings to a compatible open-source cryptography library. This is expected to require a considerable effort over time, and will be broken up into individual proposals, each of which will be accompanied by its own  specific set of goals.
```

Comment:
Was the text in square brackets supposed to be a link? The formatting seems a bit off.

Reply:
Oops that is a bit of my scratch note formatting that escaped containment. It means I was considering an alternate sentence fragment / rewording it.